src/features/flows/nRF54L15/index.ts

```diff
@@ -0,0 +1,229 @@
+        },
+    },
+];
+
+const evaluateConfig = [
+    {
+        ref: 'Hello World',
+        resources: [
+            {
+                title: 'Test with serial output',
+                description:
+                    'Open the serial terminal application and press reset on the device to print output.',
+                app: 'pc-nrfconnect-serial-terminal',
+                vComIndex: 1,
+            },
+            {
+                title: 'Documentation',
+                description:
+                    'Read the whole documentation for the Hello World application.',
+                mainLink: {
+                    label: 'Open documentation',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/samples/hello_world/README.html',
+                },
+            },
+        ],
+    },
+    {
+        ref: 'LED Button Service',
+        resources: [
+            {
+                title: 'Test the sample',
+                description:
+                    'Follow the test instructions to evaluate the sample.',
+                mainLink: {
+                    label: 'Test instructions',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_lbs/README.html#testing',
+                },
+            },
+            {
+                title: 'Documentation',
+                description:
+                    'Read the whole documentation for the Peripheral LBS application.',
+                mainLink: {
+                    label: 'Open documentation',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_lbs/README.html',
+                },
+            },
+        ],
+    },
+    {
+        ref: 'Nordic UART Service',
+        resources: [
+            {
+                title: 'Test the sample',
+                description:
+                    'Follow the test instructions to evaluate the sample.',
+                mainLink: {
+                    label: 'Open instructions',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_uart/README.html#testing',
+                },
+            },
+            {
+                title: 'Documentation',
+                description:
+                    'Read the whole documentation for the Nordic UART application.',
+                mainLink: {
+                    label: 'Open documentation',
+                    href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/peripheral_uart/README.html',
+                },
+            },
+        ],
+    },
+];
+
+const learnConfig = [
+    {
+        label: 'Developer Academy',
+        description: 'Interactive online learning platform for Nordic devices.',
+        link: {
+            label: 'Nordic Developer Academy',
+            href: 'https://academy.nordicsemi.com/',
+        },
+    },
+    {
```

Comment:
We also mentioned during the meeting to have a middle section about NCS and Zephyr here.
This needs to be added to all index.ts, and replace the IoT best practices for nRF91 devices.
Proposal:
```suggestion
    },
    {
        label: 'nRF Connect SDK and Zephyr',
        description: 'Learn about the application development in the nRF Connect SDK and Zephyr.',
        link: {
            label: 'Application development',
            href: 'https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev.html',
        },
    {
```